bus/bus.go

```diff
@@ -442,7 +442,6 @@ func (b *Bus) Handler() http.Handler {
 		"GET    /contract/:id/roots":     b.contractIDRootsHandlerGET,
 		"GET    /contract/:id/size":      b.contractSizeHandlerGET,
 
-		"GET    /hosts":                          b.hostsHandlerGETDeprecated,
```

Comment:
Since we remove `/hosts` we could rename the search endpoint to `/hosts`. That way all the host related routes are nicely organized under the same name. The `search` endpoint has always seemed a bit out of place.